### PR TITLE
feat: Explicitly grant `elasticloadbalancing:AddTags` permission 

### PR DESCRIPTION
**Pull request by acdha**
As per AWS customer notifications, the `elasticloadbalancing:AddTags` permission will be required soon for anyone calling `CreateLoadBalancer` with tags:

> On June 1, 2023, we will be adding an additional layer of security to ELB ‘Create*' API calls where API callers must have explicit access to add tags in their Identity and Access Management (IAM) policy [1]. Currently, access to attach tags was implicitly granted with access to 'Create*' APIs. … We will be allowing 'Create*' API calls with the current policy to be accepted until August 30, 2023. After this date, the 'Create*' API call will fail and return an error if the the attribute is specified and permission is not granted.

### CODE REVIEW

**Review by bryantbiggs — APPROVED** [review]

**Comment by antonbabenko** [comment]
This PR is included in version 5.21.0 :tada:

**Comment by tomelliff** [comment]
I don't think this is the equivalent of [the current upstream policy](https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.5.3/docs/install/iam_policy.json) but the translation to HCL makes it a little hard to follow.

The diff between [v2.4.3](https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.4.3/docs/install/iam_policy.json) and v2.5.3 is this statement block:

```diff
+        {
+            "Effect": "Allow",
+            "Action": [
+                "elasticloadbalancing:AddTags"
+            ],
+            "Resource": [
+                "arn:aws:elasticloadbalancing:*:*:targetgroup/*/*",
+                "arn:aws:elasticloadbalancing:*:*:loadbalancer/net/*/*",
+                "arn:aws:elasticloadbalancing:*:*:loadbalancer/app/*/*"
+            ],
+            "Condition": {
+                "StringEquals": {
+                    "elasticloadbalancing:CreateAction": [
+                        "CreateTargetGroup",
+                        "CreateLoadBalancer"
+                    ]
+                },
+                "Null": {
+                    "aws:RequestTag/elbv2.k8s.aws/cluster": "false"
+                }
+            }
+        },
```

which appears to already be in the existing policy having been added in https://github.com/terraform-aws-modules/terraform-aws-iam/commit/e1403c15285137147db2698465e794d17059c848 in https://github.com/terraform-aws-modules/terraform-aws-iam/pull/358.

Would it make more sense to instead use the raw JSON from upstream here to make it much easier to maintain this and show the direct link to upstream?
